sopel/modules/wiktionary.py

```diff
@@ -6,7 +6,8 @@
 
 https://sopel.chat
 """
-from __future__ import unicode_literals, absolute_import, print_function, division
+from __future__ import unicode_literals, absolute_import, print_function, \
+    division
```

Comment:
Please remove this style-only change. No other files in Sopel's codebase have this line-break. (The style checker specifically ignore line length, for now.)

Reply:
Sorry about all the line breaks. My IDE insists on auto-cleaning my code that way :/ I have convinced it not to anymore. Requested change coming right up